From b10f7384ae59f1df1db5f1f31cbe7eea308a4b69 Mon Sep 17 00:00:00 2001
From: Paul Scovanner <pscovanner@google.com>
Date: Tue, 11 Jun 2019 15:20:57 -0700
Subject: [PATCH 78/81] Update Security String from 08-05 to 2019-08-01
 Bug:126590667

Merged-In: a3e1c57a95d895fff372054205914bc322b025a1
(cherry picked from commit 0b6993aa3aa9053f3a762e380a8fb8f68636b932)

Change-Id: Iae0be422cc387e2b7a8310cc1a09ce90250ab84f
---
 core/version_defaults.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/version_defaults.mk b/core/version_defaults.mk
index 0e6473911..ab7c2229c 100644
--- a/core/version_defaults.mk
+++ b/core/version_defaults.mk
@@ -231,7 +231,7 @@ ifndef PLATFORM_SECURITY_PATCH
     #  It must be of the form "YYYY-MM-DD" on production devices.
     #  It must match one of the Android Security Patch Level strings of the Public Security Bulletins.
     #  If there is no $PLATFORM_SECURITY_PATCH set, keep it empty.
-      PLATFORM_SECURITY_PATCH := 2019-08-05
+      PLATFORM_SECURITY_PATCH := 2019-08-01
 endif
 
 ifndef PLATFORM_SECURITY_PATCH_TIMESTAMP
-- 
2.17.1

